oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/TokenHandler.java

```diff
@@ -115,7 +116,8 @@ public TokenHandler(ConfigurationService configurationService) {
                         new JwksService(configurationService, kms), configurationService);
         this.tokenClientAuthValidatorFactory =
                 new TokenClientAuthValidatorFactory(
-                        configurationService, new DynamoClientService(configurationService));
+                        new DynamoClientService(configurationService),
+                        new ClientSignatureValidationService(configurationService));
```

Comment:
`/home/runner/work/authentication-api/authentication-api/oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/TokenHandler.java:141: error: incompatible types: ConfigurationService cannot be converted to DynamoClientService

                        configurationService, new DynamoClientService(configurationService));`

Can't work out what the build / contract test checks are complaining about?